cli/src/commands/sandbox.rs

```diff
@@ -48,13 +50,21 @@ pub async fn handle_sandbox(matches: &ArgMatches) -> CommandResult {
 /// Lock down the current process.
 #[cfg(unix)]
 fn lock_process(matches: &ArgMatches) -> Result<()> {
-    let birdcage =
-        if matches.get_flag("strict") { Birdcage::new() } else { permissions::default_sandbox() };
+    let min_landlock_abi = matches.get_one("min-landlock-abi").map_or(1, |abi| *abi);
+    let best_effort = matches.get_flag("best-effort");
+    let strict = matches.get_flag("strict");
+
+    let birdcage = permissions::default_sandbox(strict, min_landlock_abi);
 
     // Provide additional error context.
     let mut birdcage = match birdcage {
         Ok(birdcage) => birdcage,
         #[cfg(target_os = "linux")]
+        Err(_) if best_effort => {
+            log::debug!("Landlock v{min_landlock_abi} is not supported, skipping sandbox");
+            return Ok(());
+        },
```

Comment:
Technically sandboxing can still fail when `add_exception` fails or `lock` fails. However this shouldn't be a landlock ABI error at that point so I don't think `best_effort` should apply?

There might be some scenarios we want to include into `best_effort` in the future, we could for example make ignore an error in `lock` when both seccomp and namespaces are blocked. But with our current behavior we should support pretty much every system so I don't think there's a need to relax things even more.